server/server/views/hot_event.py

```diff
@@ -0,0 +1,46 @@
+"""View for displaying hot events"""
+
+from cornice.resource import resource, view
+from pyramid.security import Allow, Everyone
+
+from ..models import model_to_dict
+from ..models.event import Event
+from ..models.event_history import EventHistory
+
+
+@resource(collection_path='/hot-event/',
+          path='/hot-event/{hotEvent_id}', renderer='json',
+          cors_origins=('http://localhost:3000',))
+class HotEventView(object):
+
+    def __init__(self, request, context=None):
+        self.request = request
+        self.context = context
+
+    def __acl__(self):
+        return [(Allow, Everyone, 'get')]
+
+    @view(permission='get')
+    def collection_get(self):
+        request = self.request
+
+        response = {
+            'hot_events': None,
+            'display_method': None
+        }
+
+        histories = EventHistory.get_all_by(request, status_id=5)
```

Comment:
You have some problem here. What if event was Hot, but later you change this status to other?
(Record at Event History with status Hot will stay)

Reply:
Yes, set status Hot you can only manually, but if the user changes some data in this event, the event status will be "New" with comment - "Wait for review by moderator". All you need it's check the status with the LAST date. Not all event history, just record with the last date